### PR TITLE
feat: formatter

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

```js
const options = {
  formatter: (value) => string;
};
```

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
[08/24] 리뷰완료
고생하셨습니다!

File: src/component/tooltip.ts [hunk 1]
```diff
@@ -91,7 +93,13 @@ export default class Tooltip extends Component {
           acc.target.height = height;
         }
 
-        acc.data.push(data);
+        acc.data.push({
+          ...data,
+          formattedValue: this.formatter
+            ? this.formatter(data.value as SeriesDataType)
+            : getValueString(data.value),
+          formatter: this.formatter,
```
Comment (jungeun-cho) [inline src/component/tooltip.ts:101]: `formattedValue`만 사용되는것같은데 formatter정보를 넣어주는 이유를 알 수 있을까요?_?

클라이언트측에서 model을 받는다고하더라도 이미 입력한 tooltipFormatter를 알고있을것같아서요~

**Review by jungeun-cho — APPROVED** [review]
[8/24] 이견없습니다. 리뷰 완료합니다.

File: stories/tooltip.stories.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import LineChart from '@src/charts/lineChart';
+import { DefaultTooltipTemplate, LineSeriesData, SeriesDataType } from '@t/options';
+import { deepMergedCopy } from '@src/helpers/utils';
+import { temperatureData, temperatureData2 } from './data';
+import { boolean, number, withKnobs } from '@storybook/addon-knobs';
```
Comment (jungeun-cho) [inline stories/tooltip.stories.ts:5]: 사용하지 않는 import 확인부탁합니다.

**Pull request merged**